SYCL/ESIMD/api/functional/functions/functions_1d_select.hpp

```diff
@@ -0,0 +1,292 @@
+//===-- functions_1d_select.hpp - Functions for tests on simd rvalue select
+//      function. ---------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// This file provides functions for tests on simd 1d select function.
+///
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "../shared_element.hpp"
+#include "common.hpp"
+
+// for std::ceil
+#include <cmath>
```

Comment:
Just to make sure - do we still need it?

Reply:
no, removed in this commit: https://github.com/intel/llvm-test-suite/pull/875/commits/7f9db3eebf5ec8ff52475f0920fad0fb0875349f